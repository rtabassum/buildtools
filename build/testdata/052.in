# Binary operators

foo=bar+baz
foo=bar-baz
foo=bar*baz
foo=bar/baz
foo=bar//baz
foo=bar%baz

# Logic operators
not  foo<bar>baz<=foo>=bar!=baz  in  foo==bar  or  baz

# Augmented assignments

foo+=bar
foo-=bar
foo*=bar
foo/=bar
foo//=bar
foo%=bar
