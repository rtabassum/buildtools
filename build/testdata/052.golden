# Binary operators

foo = bar + baz

foo = bar - baz

foo = bar * baz

foo = bar / baz

foo = bar // baz

foo = bar % baz

# If-else operator precedence
-f(1) if not a else (2 + 3)

# Logic operators
not foo < bar > baz <= foo >= bar != baz in foo == bar or baz

# Augmented assignments

foo += bar

foo -= bar

foo *= bar

foo /= bar

foo //= bar

foo %= bar
